db/migrate/20181227165901_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPosts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :posts, :user_id, :integer
```

Comment:
`add_foreign_key :posts, :user` is more robust, as it adds foreighn key at database level.